src/libtilt/projection/project_real.py

```diff
@@ -73,5 +73,5 @@ def _project_volume(rotation_matrix) -> torch.Tensor:
 
     yl, yh = padding[1, 0], -padding[1, 1]
     xl, xh = padding[2, 0], -padding[2, 1]
-    images = [_project_volume(matrix)[yl:yh, xl:xh] for matrix in rotation_matrices]
+    images = [_project_volume(matrix)[yl:yh, xl:xh] for matrix in rotation_matrices] #TODO: This can probabaly optimized using vmap
```

Comment:
https://pytorch.org/docs/stable/generated/torch.vmap.html for self

to copy to an issue